### PR TITLE
chore: add default media sources to theme editor input

### LINKED ISSUE #9: Implement default media sources
## Description

As a user, I would like the theme editor to provide default media sources that represent typical use cases. This will facilitate quicker testing and understanding of different media interactions within the video player environment.

## Acceptance Criteria

- The live preview section of the theme editor must include a pre-configured selection of default media sources.
- These default options should include typical scenarios: Live Video, Live Audio, VoD, and AoD.

### PR DESCRIPTION
**Pull request by jboix**
## Description

Resolve #9 by adding a pre-configured selection of default media sources to the theme editor's live preview section.

## Changes Made

The input field now includes a `datalist` element with default options for typical media scenarios. The `datalist` element was chosen to provide a simple and native solution for suggesting media sources, ensuring compatibility and ease of use across most browsers.

While Firefox has limited support for the `datalist` element, the decision to use it was made to simplify the implementation and utilize native HTML5 elements when possible. See the compatibility chart: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist

See the following comparison: 

| Chrome                                                                                                                          | Firefox                                                                                                                          | Safari                                                                                                                          |
|---------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
| Chrome Screenshot | Firefox Screenshot | Safari Screenshot |

## Checklist

- [x] I have followed the project's style and contribution guidelines.
- [x] I have performed a self-review of my own changes.
- [x] I have made corresponding changes to the documentation.
- [x] I have added tests that prove my fix is effective or that my feature works.

### CODE REVIEW

File: index.html
Comment (defagos) [inline index.html]:
```suggestion
            <option value="urn:rts:video:3608506">Live video (DVR)</option>
```

**Review by defagos — APPROVED** [review]

**Comment by defagos** [comment]
@jboix  I see that this repository does not check that all discussions associated with a PR have been resolved before the PR can be merged. Shouldn't you enable this feature?

**Comment by jboix** [comment]
> @jboix I see that this repository does not check that all discussions associated with a PR have been resolved before the PR can be merged. Shouldn't you enable this feature?

@defagos we normally don't approve the PR if there are open discussion, I didn't even know that was a feature. Thanks for the feedback!

**Pull request merged**
